### ISSUE TITLE
Lines near amplifier mask produce discontinuities in CCF flux

### ISSUE THREAD

**Issue by rrubenza** [Keck-DataReductionPipelines/KPF-Pipeline#701]
First, the amplifier mask is working great and really helped clean up the CCFs during the KELT-18 transit. Big boost to SNR now that CTI is not affecting line shapes.

I noticed though in the order-by-order CCFs, for order < 20 in green only, that there are many variable discontinuities in flux. @shalverson hypothesized this is due to lines near the edge of the amplifier mask (which is near peak-blaze at the center of the order) getting masked for some velocity steps and getting included in the CCF for other velocity steps. Example attached below.

From a brief discussion at the DRP meeting with @awhoward, @shalverson, @RussLaher, and @cwang2016, a simple solution would be to use the same "padding" procedure used for the edge of the detector, but to also apply it (double count) to the good "half" of the affected orders. Or, to incorporate a separate padding just for the masked orders, which uses the CCF velocity grid width to pad just the left (nearest the mask) side to remove contributions from any lines which have a chance of being doppler-shifted into the mask.

![image](https://github.com/Keck-DataReductionPipelines/KPF-Pipeline/assets/29154477/b8ddcd26-771f-4b26-936d-cfcb411dadd0)

**Comment by rrubenza** [comment]
I think this is as simple as defining `area_def = [0, 34, 2540, -500]` in the recipe if the amplifier mask in green is being applied. See line 609 of [`kpf_drp.recipe`](https://github.com/Keck-DataReductionPipelines/KPF-Pipeline/blob/da7f472d8f55f3314379093e6cd49e921110234d/recipes/kpf_drp.recipe)

I tried to test this myself but ran into errors when running the pipeline of the form
```
[KPFPipe-3][ERROR]:Exception list index out of range while invoking resume_recipe
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/keckdrpframework/core/framework.py", line 229, in execute
    action_output = pipeline.get_action(action_name)(action, context)
  File "/code/KPF-Pipeline/kpfpipe/pipelines/kpfpipeline.py", line 252, in resume_recipe
    self._recipe_visitor.visit(self._recipe_ast)
  File "/usr/local/lib/python3.6/ast.py", line 253, in visit
    return visitor(node)
  File "/code/KPF-Pipeline/kpfpipe/pipelines/kpf_parse_ast.py", line 138, in visit_Module
    self.visit(item)
  File "/usr/local/lib/python3.6/ast.py", line 253, in visit
    return visitor(node)
  File "/code/KPF-Pipeline/kpfpipe/pipelines/kpf_parse_ast.py", line 411, in visit_Assign
    self.visit(node.value)
  File "/usr/local/lib/python3.6/ast.py", line 253, in visit
    return visitor(node)
  File "/code/KPF-Pipeline/kpfpipe/pipelines/kpf_parse_ast.py", line 1067, in visit_Subscript
    self._load.append(value[sliceName])
IndexError: list index out of range
```

**Comment by rrubenza** [comment]
I hadn't, but I just changed both of the `do_query_db` to `False` and I get the same error

**Comment by rrubenza** [comment]
This should be fixed in PR #722 - the problem was actually zero flux in the masked portion of the spectrum producing the discontinuity
